package.json

```diff
@@ -14,6 +14,7 @@
     "biome_lint": "bunx @biomejs/biome lint src",
     "biome_check": "bunx @biomejs/biome check --apply src",
     "test": "jest",
+    "test1": "jest src/services/__tests__/contract.ts",
```

Comment:
why is it here?

Reply:
I was fixing that test but forgot to remove it. I can remove it